src/frontend/komponenter/Infotrygd/Infotrygd.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+import { Knapp } from 'nav-frontend-knapper';
+import { Input, SkjemaGruppe } from 'nav-frontend-skjema';
+import { Innholdstittel } from 'nav-frontend-typografi';
+
+import { byggFunksjonellFeilRessurs, Ressurs, RessursStatus } from '@navikt/familie-typer';
+
+import { useInfotrygd } from '../../context/InfotrygdContext';
+import { IInfotrygdsaker } from '../../typer/infotrygd';
+import { hentFrontendFeilmelding } from '../../utils/ressursUtils';
+import { Sakstabell } from './Sakstabell';
+
+const InfotrygdContainer = styled.div`
+    margin: 16px;
+`;
+
+const HentSakerFlex = styled.div`
+    margin-top: 64px;
+    margin-bottom: 32px;
+    display: flex;
+`;
+
+const HentSakerKnapp = styled(Knapp)`
+    margin-left: 1rem;
+    margin-top: 30px;
+    margin-bottom: auto;
+    height: 40px;
+`;
+
+export const Infotrygd: React.FC = () => {
+    const {
+        onSubmit,
+        tilgangFeilmelding,
+        settSubmitRessurs,
+        skjema,
+        sorterSakerEtterSaksnr,
+    } = useInfotrygd();
+
+    const skjemaErLåst = skjema.submitRessurs.status === RessursStatus.HENTER;
+
+    const visTabell = () => {
```

Comment:
Fint å trekke ut i egen funksjon, dette kunne også ligget inne i Sakstabell tenker jeg :)

Reply:
Dette er jo bare en verdi som oppdateres pr. render, hva blir nytten av å trekke den ut som en egen funksjon? Og hva slags rolle skal denne verdien spille i sakstabell? Tar gjerne en prat på det.

Reply:
Jeg mente bare at det var fint at dere dro det ut i en funksjon :) Også mente jeg at denne logikken kunne også ligget i komponenten, men ikke farlig :)

Reply:
Tja.. da lar vi den ligge for nå - usikker på om du mener en funksjon med skjema som parameter utenfor renderfunksjonen, eller en parameterløs funksjon i selve renderfunksjonen..

Sakstabellen tar ferdig hentede saker som input, så den forholder seg ikke til RessursStatus.HENTER.